tests/suites/1_stateful/00_copy/00_0011_distributed_copy_into_table_exection_test.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. "$CURDIR"/../../../shell_env.sh
+
+echo "set global max_threads = 1" | $MYSQL_CLIENT_CONNECT
+echo "drop table if exists products;" | $MYSQL_CLIENT_CONNECT
+echo "drop stage if exists s1;" | $MYSQL_CLIENT_CONNECT
+echo "CREATE STAGE s1 FILE_FORMAT = (TYPE = CSV);" | $MYSQL_CLIENT_CONNECT
```

Comment:
although recreated, there maybe old files in s1

https://github.com/datafuselabs/databend/issues/10564